js/views/pageNavVw.js

```diff
@@ -10,7 +10,6 @@ var __ = require('underscore'),
     adminPanelView = require('../views/adminPanelVw'),
     notificationsPanelView = require('../views/notificationsPanelVw'),
     remote = require('remote'),
-    cropit = require('../utils/jquery.cropit'),
```

Comment:
You can also remove these lines:

countryListView = require('../views/countryListVw'),
    currencyListView = require('../views/currencyListVw'),
    languageListView = require('../views/languageListVw'),